test/assembly/rules.bzl

```diff
@@ -0,0 +1,73 @@
+#
+# Copyright (C) 2021 Grakn Labs
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+#
+
+def server_java_test(name, server_mac_artifact, server_linux_artifact, server_windows_artifact, deps = [], classpath_resources = [], data = [], **kwargs):
```

Comment:
When we want to test Grakn we'll have to use `server_java_test`, but when we want to test the integration between Grakn and Console we'll have to use `console_java_test`? The name feels inconsistent. What if we just expose one rule, `grakn_java_test` which may optionally take Console artifacts?

Reply:
that sounds like a good idea.